commit 42156d2b5ac797e5f28f8d0d38b691053f3f6fc7
Author: Colin Guthrie <colin@mageia.org>
Date:   Fri Oct 17 14:43:18 2014 +0200

    launch: Avoid specifically starting PA and rely on autospawn/socket activation
    
    This --start is patched out in several downstreams to allow users to easily
    disable PA by simply disabling autospawn.
    
    If autospawn is enabled, then the first pactl command will start it and if not
    it will fail and the script will exit.
    
    When switching to systemd socket activation, we very much do not want to
    start PA manually here. We could replace it with a
      systemctl --user start pulseaudio
    but really it just makes sense to rely on the socket activation as this
    should apply equally to non-systemd setups which use PA's own autospawn.

diff --git a/src/daemon/start-pulseaudio-x11.in b/src/daemon/start-pulseaudio-x11.in
index 620e50f..63ed740 100755
--- a/src/daemon/start-pulseaudio-x11.in
+++ b/src/daemon/start-pulseaudio-x11.in
@@ -19,8 +19,6 @@
 
 set -e
 
-@PA_BINARY@ --start "$@"
-
 if [ x"$DISPLAY" != x ] ; then
 
     @PACTL_BINARY@ load-module module-x11-publish "display=$DISPLAY" > /dev/null
